src/load.ts

```diff
@@ -352,6 +357,38 @@ export function loadFromPath(
   }
 }
 
+export async function getLatestVersion(
+  packageName: string,
+  log: LogFunction = () => {}
+): Promise<string> {
+  const customRegistry = getCustomRegistry(log);
+  let res: AxiosResponse;
+  if (customRegistry) {
+    res = await axiosGet(`${customRegistry.replace(/\/$/, '')}/${packageName}`, {
+      responseType: 'json'
+    });
+  } else {
+    try {
+      res = await axiosGet(`https://packages.fhir.org/${packageName}`, {
+        responseType: 'json'
+      });
+    } catch (e) {
+      // Fallback to trying packages2.fhir.org
+      res = await axiosGet(`https://packages2.fhir.org/packages/${packageName}`, {
+        responseType: 'json'
+      }).catch(() => {
+        // If the fallback failed too, just throw the original error
+        throw e;
+      });
+    }
+  }
+  if (res?.data?.['dist-tags']?.latest?.length) {
+    return res.data['dist-tags'].latest;
+  } else {
+    throw new LatestVersionUnavailableError(packageName, customRegistry);
+  }
+}
+
```

Comment:
I agree... so I'd suggest either catching the thrown error at a place that doesn't break the loop or modifying the function to return null instead of throwing.

Reply:
Looking at the code and the logs I am getting now, I might have been wrong about this. It seems like things are loading properly, so disregard this comment!